packages/wonder-blocks-form/components/checkbox-group.test.js

```diff
@@ -66,13 +66,13 @@ describe("CheckboxGroup", () => {
     it("calls onChange for each new selection", () => {
         // a is clicked
         const a = group.find(Choice).at(0);
-        const aTarget = a.find("ClickableBehavior");
+        const aTarget = a.find("clickable_behavior_ClickableBehavior");
```

Comment:
I don't understand these changes -- they're also in the snapshot below for some of the typography components!? I tried searching all around and I can't figure out where this change may have come from.